.all-contributorsrc

```diff
@@ -0,0 +1,172 @@
+{
+  "projectName": "kcdraidgroup",
+  "projectOwner": "kcdraidgroup",
+  "repoType": "github",
+  "repoHost": "https://github.com",
+  "files": [
+    "README.md"
+  ],
+  "imageSize": 100,
+  "commit": false,
+  "commitConvention": "none",
+  "skipCi": true,
+  "contributorsPerLine": 7,
+  "types": {
+    "raider": {
+      "symbol": "⚔",
+      "description": "Raider",
+      "link": "#raids-crossed_swords"
+    }
```

Comment:
This is where the custom type is configured.  The link currently point to [the Raids section of the README](https://github.com/kcdraidgroup/kcdraidgroup#raids-crossed_swords), because I couldn't think of anywhere better to send them.

Feedback on the any of it is welcome.

Reply:
Maybe to the invite of the kcd discord, since that's where it's organised :)